### DISCUSSION TITLE
Provide install from Github instructions

### DISCUSSION THREAD

**Discussion by wdduncan** [EvgSkv/logica#394] [Ideas]
Can you update the README to include instructions for installing `logica` from the github repository? This will help users wanting to use the most recent version.

**Comment by EvgSkv** [comment]
@wdduncan that's a good question.

README says:

> Alternatively, you can clone GitHub repository:

```
git clone https://github.com/evgskv/logica
cd logica
./logica - print Greet <<<'Greet(greeting: "Hello world!")'
```

I thought this would be sufficient for getting the latest version and experimenting with it. This is how I work with it myself. I.e. I don't _install_ it into the system. I simply have it in a folder and run it from that folder, writing scripts into it. Does this not work for your workflow?

**Reply by wdduncan** [reply]
I do something like this too, but it is a bit clunky. I create a soft link to my local `logica` repo, and then access the libraries via the soft link. This allows me to import the repo version of `logica_lib.RunPredicateFromString` instead of the one created using `pip install logica`.  
I am experimenting with creating a library for using `logica` with ontologies. It would be nice to be able to do something like `pip install -e .` or point folks to the github repo to install.

**Comment by EvgSkv** [comment]
Oh yeah, this makes sense. I'll need to learn a little about `setup.py` and test the feature. I exect that I'll be able to create this option (which seems like it's supposed to be very easy) soon!
